redis/src/main/scala/zio/redis/api/Streams.scala

```diff
@@ -0,0 +1,277 @@
+package zio.redis.api
+
+import java.time.Duration
+
+import zio.redis.Input._
+import zio.redis.Output._
+import zio.redis._
+import zio.{ Chunk, ZIO }
+
+trait Streams {
+  import Streams._
+  import XGroupCommand._
+
+  final def xAck(key: String, group: String, id: String, ids: String*): ZIO[RedisExecutor, RedisError, Long] =
```

Comment:
Add some scala doc either now or in the follow up with tests.

Reply:
I will add it with tests, as there can be some API changes while writing tests.